### ISSUE TITLE
feature request: more/better tiling

### ISSUE THREAD

**Issue by plagasul** [linuxmint/Cinnamon#4897]

**Comment by rpbaptist** [comment]
With the advent of widescreen monitors this would be a very welcome addition and it's the only thing I am missing in Cinnamon. Even having the option to tile to a third or a quarter of the screen with keyboard shortcuts would be incredibly helpful.

**Comment by mainmachine** [comment]
I will say I find the tiling in Cinnamon to be really good compared to other DE's, but on a 4K display I can see where you might want to do even more granular tiling...

**Comment by rpbaptist** [comment]
@mainmachine I agree. It is why I choose Cinnamon even after moving away from Linux Mint. As I said, it's the only thing lacking for me and makes me sometimes consider investing time into configuring i3, but to be honest, that would be the only reason.
